### PR TITLE
Fix resource issues with new csproj format, and add build server

### PR DESCRIPTION
**Pull request by glennawatson**
This will make sure that commits and pull requests compile correctly.

### CODE REVIEW

**Comment by glennawatson** [comment]
Doesn't fix the existing deploy docs github action, but now at least you tell if you get a compile issue.

File: .github/workflows/dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      run: |
+        curl https://vatsys.sawbe.com/downloads/vatSysSetup.zip -o vatsys.zip
+        7z x vatsys.zip
+        mkdir ${{ runner.temp }}\vatsys
+        msiexec /a vatSysSetup.msi /qb TARGETDIR=${{ runner.temp }}\vatsys
+
+    - name: Build
+      run: msbuild /t:build /m /v:minimal /p:VatSysDir='${{ runner.temp }}\vatsys'
+
+    #- name: Test
```
Comment (glennawatson) [inline .github/workflows/dotnet.yml:45]: No unit tests to run, so no point running this.

File: OzStripsGUI/OzStripsGUI.csproj [hunk 2]
```diff
@@ -12,6 +12,6 @@
 		<PackageReference Include="SocketIOClient" Version="3.1.1" />
 	</ItemGroup>
 	<ItemGroup>
-		<Reference Include="$(MSBuildProgramFiles32)\vatSys\bin\vatSys.exe "/>
+		<Reference Include="$(VatSysDir)\bin\vatSys.exe " />
```
Comment (glennawatson) [inline OzStripsGUI/OzStripsGUI.csproj:15]: This is where the build magic comes in, by default it'll pick up the program files 86 directory, on the build server it'll pick up where the vatsys download has happened 

Nice thing also in theory this should handle the case where vatsys updates, and things may have changed.

File: OzStripsGUI/OzStripsGUI.csproj [hunk 3]
```diff
@@ -4,14 +4,22 @@
 		<UseWindowsForms>true</UseWindowsForms>
 		<AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
 		<PlatformTarget>x86</PlatformTarget>
+		<ApplicationIcon>ozstrips.ico</ApplicationIcon>
```
Comment (glennawatson) [inline OzStripsGUI/OzStripsGUI.csproj:7]: This will stop a exception happening when loading into vatsys.

**Review by maxrumsey — APPROVED** [review]
Thanks Glenn!

**Pull request merged**